rs/nns/governance/canister/governance.did

```diff
@@ -809,6 +921,18 @@ type ProposalData = record {
   wait_for_quiet_state : opt WaitForQuietState;
   executed_timestamp_seconds : nat64;
   original_total_community_fund_maturity_e8s_equivalent : opt nat64;
+
```

Comment:
```suggestion
The total voting power of neurons that could participate in deciding this proposal.
```
(I would first add the highest level intuition of what it is before the more detailed explanation)

Reply:
Ok.

Reply:
Done.